### PR TITLE
updating AssetResource and AssetMapping docs

### PR DESCRIPTION
**Pull request by agrakhov**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@robertengaxway, please verify the content before I edit. Thanks.

File: content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md [hunk 1]
```diff
@@ -95,49 +95,54 @@ Where `asset.json` contains the following content:
...
+`AssetResources` are the business value that the asset is wrapping and can be SDKs, scripts, APIs, etc. Similar to an `APIServiceRevision`, the API Server is not opinionated about what the AssetResource is. There are two ways to create an `AssetResource`:
+
+* Manual, by creating a new `AssetResource` in the `Asset` scope directly
+* Automated **(preferred)**, by creating an `AssetMapping` that contains links to the desired resources. Creating an `AssetMapping` will trigger the creation of a new `AssetResource` in the `Asset` scope.
 
-Create the AssetResource in the scope of the previously created asset. In this example, the asset will be grouped in the stage that was created previously:
+In this example, we will create an `AssetMapping` in the scope of the previously created `Asset`. The asset will be grouped in the stage created previously. Also, assuming you are using resources and corresponding files (`api-service-created.json`, `api-service-revision-created.json`, `api-service-instance-created.json`) generated in the previous step ([Register APIs using the CLI](/docs/manage_service_registry/service_integrate-api_cli)):
```
Comment (robertengaxway) [inline content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md:118]: I would suggest changing the stage wording to:
"The asset can be optionally grouped in a stage which was created previously."

File: content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md [hunk 2]
```diff
@@ -159,11 +164,11 @@ axway central apply -f asset-changed.json
 
 To release an asset, you must create a release tag for the asset. The Asset Catalog enforces sematic versioning, so when you create a release tag you must specify whether it is a major, minor or patch update. The Amplify platform will automatically calculate the semantic version of the asset based on historic release times that have been applied. You can select how the release version number is incremented by selected one of the following values: `major`, `minor`, or `patch`. If the current version number is 1.0.1, then the version is incremented to:
```
Comment (robertengaxway) [inline content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md:165]: typo, "semantic" versioning.

**Review by robertengaxway — COMMENTED** [review]
I reviewed and added some suggestions and found 1 typos

**Comment by lbadenhop** [comment]
@agrakhov I will review and merge once you have made changes @robertengaxway suggested.

File: content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md (hunk 1 above)
Comment (agrakhov) [reply content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md]: updated

File: content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md (hunk 2 above)
Comment (agrakhov) [reply content/en/docs/manage_asset_catalog/asset_integrate_api_cli.md]: fixed

File: static/samples/central/create_asset.yaml [hunk 3]
```diff
@@ -1,80 +1,94 @@
...
-    apiServiceInstance: apisvcinst2
-    stage: production
-    asset: testasset
-  assetMappingTemplate: assettemplate
-
+    stage: demostage
+    apiService: management/demoenv1/demosvc1
+    apiServiceRevision: management/demoenv1/demosvc1rev1
+    # optional:
+    apiServiceInstance: management/demoenv1/demosvc1rev1inst1
```
[the remark this answers is not in the document]
Comment (agrakhov) [reply static/samples/central/create_asset.yaml:110]: a new revision and corresponding mapping added

**Review by robertengaxway — APPROVED** [review]
I approve

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**